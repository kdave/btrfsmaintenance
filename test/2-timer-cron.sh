#!/bin/bash

# Test if the cron timers are enabled as per the config file

. $(dirname $(realpath "$0"))/utils

load_config "sysconfig.btrfsmaintenance.testall"

timer_setup "cron"
systemctl list-unit-files | grep btrfs
echo daily:
ls /etc/cron.daily | grep btrfs
echo weekly:
ls /etc/cron.weekly| grep btrfs
echo monthly:
ls /etc/cron.monthly | grep btrfs

timer_reset "cron"
systemctl list-unit-files | grep btrfs
ls /etc/cron.daily | grep btrfs
ls /etc/cron.weekly| grep btrfs
ls /etc/cron.monthly | grep btrfs

unload_config
