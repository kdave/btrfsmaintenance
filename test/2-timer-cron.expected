btrfsmaintenance-refresh.path              disabled
btrfs-balance.service                      static
btrfs-defrag.service                       static
btrfs-scrub.service                        static
btrfs-trim.service                         static
btrfsmaintenance-refresh.service           static
btrfs-balance.timer                        disabled
btrfs-defrag.timer                         disabled
btrfs-scrub.timer                          disabled
btrfs-trim.timer                           disabled
daily:
weekly:
btrfs-balance
btrfs-scrub
monthly:
btrfs-defrag
btrfs-trim
btrfsmaintenance-refresh.path              disabled
btrfs-balance.service                      static
btrfs-defrag.service                       static
btrfs-scrub.service                        static
btrfs-trim.service                         static
btrfsmaintenance-refresh.service           static
btrfs-balance.timer                        disabled
btrfs-defrag.timer                         disabled
btrfs-scrub.timer                          disabled
btrfs-trim.timer                           disabled
